cmd/broker/broker_suite_test.go

```diff
@@ -137,6 +143,23 @@ func NewBrokerSuiteTest(t *testing.T, version ...string) *BrokerSuiteTest {
 	return NewBrokerSuiteTestWithConfig(t, cfg, version...)
 }
 
+func NewBrokerSuitTestWithMetrics(t *testing.T, version ...string) *BrokerSuiteTest {
```

Comment:
I'd add the handler to the router in all tests, why not. In such case you don't need to create another constructor